### ISSUE TITLE
Dec 18 meeting

### ISSUE THREAD

**Issue by nicolo-ribaudo** [babel/notes#105]
@babel/babel feel free to add items to the agenda!

- [ ] How we should move forward with [`@babel/preset-modules`](https://github.com/babel/preset-modules)?
- [ ] `estree` parser plugin policy (@kaicataldo)
- [ ] Tuples&Records parser support (@nicolo-ribaudo)
    - Boolean flags vs enum for the syntax variant
- [ ] Module attributes parser support (@nicolo-ribaudo)
    - Mandatory `version` option?
- [ ] PR: _"regenerator-runtime" import now can be controlled through include/exclude option_ (https://github.com/babel/babel/pull/10768)
- [ ] Babel 8 Release Plan issues
    - [ ] Disable submodule exports for packages with a single `lib/index.js` (https://github.com/babel/babel/pull/10850#issuecomment-564672825) (@JLHwung)
    - [ ] Consider moving root AMD/UMD/SystemJS options to be plugin/preset options (https://github.com/babel/babel/issues/5473) - Taken from the [`Babel 7.next`](https://github.com/babel/babel/milestone/14) milestone, discussed in #104 but postponed to this meeting (@JLHwung)
    - [ ] Do not change env target default when browserslists config presents (https://github.com/babel/babel/issues/9962) (@JLHwung)
    - [ ] Add mandatory `version` option to the decorators plugins, and merge the two plugins in `@babel/parser`. (@nicolo-ribaudo)
    - [ ] Don't use an ad-hoc CJS format to transpile Babel helpers (https://github.com/babel/babel/issues/10746#issuecomment-561510965) (presented by @nicolo-ribaudo on behalf of @Andarist)
- [ ] Prepare v7.8.0 (@nicolo-ribaudo)
- [ ] Moving away from `make` (https://github.com/babel/babel/issues/10889) (@kaicataldo @nicolo-ribaudo)
- [ ] Funding status update

**Comment by xtuc** [comment]
> Module attributes parser support
- The syntax might change, see https://github.com/tc39/proposal-module-attributes/issues/12, it's early in the TC39 process. Should we support both the `as` and the `type: "f"` syntax under a flag?
- As discussed with @nicolo-ribaudo already, I would reject any attributes that is not `type` to avoid plugins already relying on them.
- Once the support for the syntax lands in Babel, I want to implement a POC in webpack to test out the semantics of the proposal. I don't think we can implement any of them in Babel itself.

**Comment by hzoo** [comment]
- [ ] Maybe if anyone has further thoughts on how we should move forward with https://github.com/babel/preset-modules? Maybe should do another/separate chat with @developit about it?

**Comment by nicolo-ribaudo** [comment]
I have added `@babel/preset-modules` as the first topic so that @developit isn't forced to listen to all the unrelated things :stuck_out_tongue:

**Comment by rajasekarm** [comment]
Update team section in babel website.
